tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -226,7 +226,7 @@ def __init__(self, context):
     self._handlers = None
     self.readers = {}
     self.run_paths = None
-    self._configs = None
+    self._configs = {}
```

Comment:
I know it's tiny but could we put this change in #1418 with the rest of the projector_plugin changes as well?  That way this PR isn't touching anything outside tensorboard/compat